README.md

```diff
@@ -304,6 +304,19 @@ address("/price/usd/gbp")
   .post({foo:"bar"})
 ```
 
+### Adding timeout
+When requesting a resource address will by default wait 30 seconds for a response from the resource, before responding with a ```408 Request Timeout``` error. By calling the ```timeout``` method on the request, it is possible to increase or decrease this timeout period. The following example shows how to increase the timeout from the default 30 seconds to 60:
```

Comment:
We should change the text here to say `wait at least` rather than just `wait`, since the timeout detection isn't always accurate – if there's something holding up the main thread for instance we may well be waiting longer. It's kind of an academic distinction, but still.

Reply:
Let's also change `before responding with` to `before returning`, it reads better.

Reply:
Also there's no need to use triple backticks when inlining quotes, single backticks will do fine.